lib/pipeline.js

```diff
@@ -37,28 +35,6 @@ class PipelineModel extends BaseModel {
     }
 
     /**
-     * Format the scm url to include a branch and make case insensitive
-     * @method formatScmUrl
-     * @param  {String}     scmUrl Github scm url
-     * @return {String}            Lowercase scm url with branch name
-     */
-    formatScmUrl(scmUrl) {
```

Comment:
Looks like it was moved there: https://github.com/screwdriver-cd/screwdriver/pull/88/files#diff-1eb6375f360812e82460d42f926123b1R14